### PR TITLE
Github action added for building the centos base image for ibm-powervs-block-csi-driver

### LINKED ISSUE #145: Explore using GitHub action for cross building the centos based image
**Is your feature request related to a problem?/Why is this needed**
In few case, have seen GH actions are faster for cross building docker images, lets explore for building centos based images

/feature

**Describe the solution you'd like in detail**
Reference: https://rpsene.wordpress.com/2021/09/09/cicd-building-multi-arch-container-images-with-github-actions/

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Madhan-SWE**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Added a GitHub Action on Push event to build and push multi arch images to the GitHub packages.

**Which issue(s) this PR fixes**:

Fixes #145 

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: .github/workflows/docker-build.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+      # Action reference: https://github.com/docker/build-push-action
+      - name: Build container
+        uses: docker/build-push-action@v2
+        with:
+          context: ./build/base
+          platforms: linux/amd64,linux/ppc64le
+          target: centos-base
+          push: true
+          tags: ghcr.io/${{ steps.repository_owner.outputs.lowercase }}/ibm-powervs-block-csi-driver-centos-base:V8.0
```
Comment (Madhan-SWE) [inline .github/workflows/docker-build.yml:58]: @mkumatag is it okay to hardcode the base image version?

Comment (Madhan-SWE) [reply .github/workflows/docker-build.yml:58]: The value is hardcoded here: https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/blob/abfbe652e38e5a0562c70ce1af517a31a07d7c9a/.github/workflows/docker-build.yml#L50

Comment (mkumatag) [reply .github/workflows/docker-build.yml:58]: I'm okay having it here as well, other option would be like reading from some file.

Comment (mkumatag) [reply .github/workflows/docker-build.yml:58]: can you rename rename the tag to `v8.0.0`(lowercase V)

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-build.yml (hunk 1 above)
Comment (Madhan-SWE) [reply .github/workflows/docker-build.yml]: fixed

**Review by mkumatag — APPROVED** [review]
/retest
/lgtm
/approve

**Pull request merged**